lighthouse-cli/bin.ts

```diff
@@ -151,8 +152,8 @@ function saveResults(results: Results, artifacts: Object, flags: Flags) {
         return innerPromise.then((_: Results) => Printer.write(results, outputType, outputPath));
       }, Promise.resolve(results));
     } else {
-      const outputPath =
-          flags.outputPath || `${resolvedPath}.report.${typeToExtension(flags.output)}`;
+      const outputPath = flags.outputPath ||
```

Comment:
this was actually totally breaking vscode-insiders syntax highlighting :) filed an issue about it